src/pages/Verification/Confirm.tsx

```diff
@@ -0,0 +1,96 @@
+import { useEffect, useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { Typography, CircularProgress } from '@mui/material';
+import { makeStyles } from '@mui/styles';
+import axios, { AxiosResponse } from 'axios';
+import { BASE_URL } from '@/config/default';
+import { selectToken } from "@/redux/userSlice";
+import { useSelector } from "react-redux";
+import { RootState } from '@/redux/store';
+
+
+interface AuthResponse {
+  data?: any;
+  error?: string;
+  message?: string;
+}
+
+const useStyles = makeStyles(() => ({
+  root: {
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+    justifyContent: 'center',
+    minHeight: '100vh',
+  },
+  message: {
+    marginTop: "2px",
+  },
+}));
+
+function Confirm() {
+  const classes = useStyles();
+  const navigate = useNavigate();
+  const [confirmationStatus, setConfirmationStatus] = useState('pending');
```

Comment:
I would advise using a separate type to specify the confirmationStatus instead of a string literal like this. It would avoid potential typo bugs. Something like `status = "success" | "error"`  might be better.